src/main/java/com/flexrate/flexrate_back/member/dto/SignupResponseDTO.java

```diff
@@ -0,0 +1,16 @@
+package com.flexrate.flexrate_back.member.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+
+/*
+ * 회원가입 성공 시 반환하는 DTO
+ * @since 2025.04.28
+ * @author 윤영찬
+ */
+@Getter
+@Builder
+public class SignupResponseDTO {
+    private Long userId;
+    private String email;
+}
```

Comment:
기존 코드들을 참고하시면, DTO들에는 record를 사용하고있어요!
record는 불변 객체로, 생성 후 내부 상태를 변경할 수 없어 데이터의 일관성을 보장해주고, 다중 스레드 환경에서 안전하게 사용할 수 있도록 해줘용.
또한 getter도 자동으로 생성해주니 @Getter를 선언하지 않아도 돼요.
컨벤션을 맞출수있도록 변경해주세요!


```suggestion
@Builder
public record SignupResponseDTO (
        Long userId,
        String email
) {}
```